src/applications/common/schemaform/fields/AutosuggestField.jsx

```diff
@@ -14,6 +14,13 @@ function getInput(input, uiSchema, schema) {
 
   if (typeof input !== 'object' && input) {
     const uiOptions = uiSchema['ui:options'];
+    // When using this field in an array item, editing the item will throw an error
+    //  if there uiOptions.label is undefined (as when we queryForResults), so we
+    //  have to have this safety valve
```

Comment:
Not sure this comment block is needed, strictly speaking, but it seemed like a specific enough case to warrant explanation.